rdtools/normalization.py

```diff
@@ -602,13 +586,16 @@ def trapz_aggregate(time_series, target_frequency, max_timedelta=None):
         max_interval_nanoseconds = max_timedelta.total_seconds() * 10.0**9
 
     # in x*hours
-    trap_sum = (values[1:] + values[:-1]) / 2 * t_diffs / 10**9 / 3600.0
+    if series_type is 'instantaneous':
+        series_sum = (values[1:] + values[:-1]) / 2 * t_diffs / 10**9 / 3600.0
```

Comment:
Nit, `t_diffs * 10**9 / 3600.0` is used twice, so might as well factor it out into `t_diffs_hours` or something